docs/specifications/tests/DNSSEC-TP/dnssec15.md

```diff
@@ -0,0 +1,173 @@
+# DNSSEC15: Existence of CDS and CDNSKEY
+
+## Test case identifier
+**DNSSEC15**
+
+## Objective
+
+CDS and CDNSKEY record types are defined in [RFC 7344] and [RFC 8078].
+Both record types are optional in a zone. The objective of this test
+case is to verify that they are correctly set-up, if included in the
+zone.
+
+If a CDS record is included in the zone, the corresponding CDNSKEY
+record should also be included ([RFC 7344][RFC 7344, section 4],
```

Comment:
Why not include "section 4" in the link text?

Reply:
To make the displayed link being shorter. They stick out too much.